ussp-api/utm-adapter.yaml

```diff
@@ -479,29 +479,92 @@ paths:
       tags:
         - geo-zone
 
-  /crewed_traffic_information:
+  /traffic_information:
```

Comment:
Can I request that we keep crewed traffic it is more clear and explicit

Reply:
Hi @jokalode , thanks for this, the thing is that only crewed information will be present at this interface, for un-crewed information we will use network RID, maybe I am missing something.

Reply:
Network remote ID is pretty useless for receiving traffic information.

Reply:
I would like to discuss why you think NetRID is useful for traffic information. We have conducted many trials re NetRID for a "limited time" traffic querying. The idea is to use Net RID endpoints to get NetRID traffic information and use this for exclusively crewed traffic. From the beginning the end-points were separated, especially given that many times there is no way to detect small drones, we can create a new endpoint that gives both the traffic information. Lets talk about this in the next call.